doc/security/specs/attestation/_index.md

```diff
@@ -0,0 +1,879 @@
+*   `ROM_EXT`: ROM Extension. Stored in flash and signed by the Silicon Creator.
+*   Owner boot stages. This document uses two stages as an example. The Silicon
+    Owner is free to choose other boot configurations.
+    *   `BL0`: Bootloader. Signed by the Silicon Owner.
+    *   `KERNEL`: Signed by the Silicon Owner.
+
+## Attestation Flows
+
+Device attestation is rooted in two asymmetric keys named Creator Identity and
+Owner Identity. The [Asymmetric Keys](#asymmetric-keys) section provides details
+on the cryptographic properties of these keys.
+
+<!-- TODO: Link to Identities and Root Keys doc -->
+
+The **[Creator Identity](#)** is generated at manufacturing time and it is
+endorsed by the Silicon Creator PKI. This key is used to sign certificates
+endorsing the authenticity of the physical device, the ROM and ROM\_EXT
+configuration (see [Creator custom Extension](#creator-custom-extension) for
+more details), as well as the Owner Identity public key.
+
+<!-- TODO: Link to Identities and Root Keys doc -->
+
+The **[Owner Identity](#)** is generated at ownership transfer time, or whenever
+there is a change to the BL0 software binding tag. The Owner Identity is used to
+attest the owner and BL0 configuration, as well as an attestation key used by
+the Kernel. This key is endorsed by the Creator Identity, but can also be
+endorsed by the Silicon Owner PKI. Endorsement of the Owner Identity with the
+Owner’s PKI, is covered in detail in the [Owner Personalization](#) process
+described in the provisioning specification.
+
+When using a Silicon Owner PKI, the Owner is expected to maintain a device
+registry with Creator Identity to Owner Identity certificate mappings to handle
+certificate revocations issued by the Silicon Creator.
+
+Application specific attestation keys are expected to be endorsed by the Owner
+Identity and managed by the Kernel. The format of the attestation certificate is
+outside the scope of this document.
+
+### Certificate Chains
+
+There are two possible attestation chains. Creator based attestation provides a
+chain to the Creator PKI, which is useful when testing the authenticity of the
+device. After ownership transfer, the Silicon Owner may opt to switch to its own
+PKI by endorsing the Owner Identity. On-Device certificates are stored in a
+flash region accessible to all boot stages including the kernel stage.
+Additional storage is required to store the Owner Identity certificate endorsed
+by the Silicon Owner PKI.
+
+The kernel may opt to extend the attestation chain based on application level
+requirements. See also the [Privacy Considerations](#privacy-considerations)
+section for specific cases in which extending the attestation chain is
+recommended.
+
+<table>
+  <tr>
+    <td>
+      <img src="attestation_fig1.svg" width="" alt="fig1" style="width: 800px;">
+    </td>
+  </tr>
+  <tr>
+    <td>Figure: PKI and Device Certificates</td>
+  </tr>
+</table>
```

Comment:
nit: This could be a `figure` shortcode, but this is fine.

Reply:
It'd be good to support this for consistency. I will add it to my TODOs.